int64 start
string outputRawFilePath
string outputDftFilePath
---
bool success
